packages/components/src/styles/display/display.stories.ts

```diff
@@ -38,11 +38,10 @@ export const Default = {
 };
 
 /**
- * Use the `&--size-*` classes for alternative appearances.
-
- * - `sd-display--size-4xl`: 4xl can be used as an alternative
- * - `sd-display--size-3xl`: 3xl can be used as an alternative
- * - xl is the default display size
+ * Use the `sd-display` classes for alternative appearances:
+ * - `sd-display--size-4xl`
+ * - `sd-display--size-3xl`
+ * - xl is the default size
```

Comment:
This is related to a bug [issue num. 1355](https://github.com/solid-design-system/solid/issues/1355) and [issue num. 1412](https://github.com/solid-design-system/solid/pull/1412) since the 4XL should be the default value. since in figma this is the current order and the fix in code has been addressed I believe it can stay this way for now.